tests/validation_tests/test_pre_checks.py

```diff
@@ -209,6 +249,138 @@ def valid_workbook_round_four():
             }
         ),
         "8 - Review & Sign-Off": pd.DataFrame(
+            {
+                0: ["", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""],
+                1: [
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
```

Comment:
Do we want these to be pulled out into a `mock_data.py` module again? This clutters the test module quite a lot